app/waf/src/controllers/adc.controller.ts

```diff
@@ -31,6 +36,9 @@ export class AdcController {
     public adcRepository: AdcRepository,
     @repository(AdcTenantAssociationRepository)
     public adcTenantAssociationRepository: AdcTenantAssociationRepository,
+    @inject(RestBindings.Http.CONTEXT)
+    private reqCxt: RequestContext,
```

Comment:
If we associate request context with controller, how do we switch the context for different request? Is it one context instance, or different instances?

Reply:
The controller instance is created for each request(request context).

Reply:
OK, got it.